lib/utils/telemetry/generatePayload.js

```diff
@@ -96,6 +100,13 @@ const getServiceConfig = ({ configuration, options }) => {
       .filter(Boolean),
     resources,
   };
+
+  // We want to recognize types of constructs from `serverless-lift` plugin if possible
+  if (plugins.includes('serverless-lift') && _.isObject(configuration.constructs)) {
+    result.constructs = Object.values(configuration.constructs).map(({ type }) => ({ type }));
```

Comment:
In general looks good, but we have a convention of being bulletproof here against any edge cases (any invalid configuration constructs). While now, if someone configures one of the `constructs` as `null`, this will dirty crash.

Reply:
That's a great point, I'll make sure we don't crash in such cases